src/android/StatusBar.java

```diff
@@ -227,20 +236,22 @@ private void setStatusBarBackgroundColor(final String colorPref) {
     }
 
     private void setStatusBarTransparent(final boolean transparent) {
-        if (Build.VERSION.SDK_INT >= 21) {
+        this.transparent = transparent;
             final Window window = cordova.getActivity().getWindow();
             if (transparent) {
                 window.getDecorView().setSystemUiVisibility(
                         View.SYSTEM_UI_FLAG_LAYOUT_STABLE
                                 | View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN);
-                window.setStatusBarColor(Color.TRANSPARENT);
+
+                if (Build.VERSION.SDK_INT >= 21) {
```

Comment:
Why did this SDK version check get moved inside here? 
Does the other stuff also work on earlier versions?

Reply:
As far as i've checked `setSystemUiVisibility` has been added in API 11 https://developer.android.com/reference/android/view/View.html#setSystemUiVisibility(int) so it seems it should work everywhere